### PR TITLE
[DE] Update `Tournaments`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Currently, I'm translating `History of osu!/2009` but this article is quite long and difficult to translate, so I thought of working on smaller updates in between.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Tournaments/de.md [hunk 1]
```diff
@@ -440,7 +441,7 @@ Nicht offizielle Turniere/Wettkämpfe, die von den Communities veranstaltet werd
 | Ulat Bulu Cup 2019 | 26.07.2019 | 31.08.2019 | Jidank Hamil | Susu Vanilla Cimory | Budak Cygames |
 | [Ulat Bulu Cup 2020](UCUP/2020) | 02.06.2020 | 12.07.2020 | awan fanclub | Press 3 Fs to pay respects | terserah |
 | [Ulat Cup 2021](UCUP/2021) | 13.09.2021 | 31.10.2021 | Rushia | Surtr S3 | L9 Hamster |
-| [Ulat Cup 2022](UCUP/2022) | 2021-09-13 | 2021-10-31 | ::{ flag=TH }:: [chests](https://osu.ppy.sh/users/14806365) | ::{ flag=ID }:: [BlankTap](https://osu.ppy.sh/users/10137131) | [LouisZpro](https://osu.ppy.sh/users/6231292) |
+| [Ulat Cup 2022](UCUP/2022) | 30.09.2022 | 11.12.2022 | ::{ flag=TH }:: [chests](https://osu.ppy.sh/users/14806365) | ::{ flag=ID }:: [BlankTap](https://osu.ppy.sh/users/10137131) | [LouisZpro](https://osu.ppy.sh/users/6231292) |
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/de.md:444]: TODO: Fix date in English article

File: wiki/Tournaments/de.md [hunk 2]
```diff
@@ -49,6 +44,7 @@ Offizielle Turniere, die vom osu!-Team organisiert werden.
 | [osu!taiko World Cup 2021](TWC/2021) | 06.03.2021 | 11.04.2021 | ::{ flag=JP }:: Japan | ::{ flag=KR }:: Südkorea | ::{ flag=BR }:: Brasilien |
 | [osu!taiko World Cup 2022](TWC/2022) | 12.03.2022 | 24.04.2022 | ::{ flag=JP }:: Japan | ::{ flag=KR }:: Südkorea | ::{ flag=ID }:: Indonesien |
 | [osu!taiko World Cup 2023](TWC/2023) | 18.03.2023 | 30.04.2023 | ::{ flag=JP }:: Japan | ::{ flag=TW }:: Taiwan | ::{ flag=DE }:: Deutschland |
+| [osu!taiko World Cup 2024](TWC/2024) | 16.03.2024 | 28.04.2024 | *TBD* | *TBD* | *TBD* |
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/de.md:47]: Not sure if these should be translated 🤔

File: wiki/Tournaments/de.md [hunk 3]
```diff
@@ -88,7 +84,7 @@ Offizielle Turniere, die vom osu!-Team organisiert werden.
 | [osu!mania 4K World Cup 2022](MWC/2022_4K) | 07.07.2022 | 18.09.2022 | ::{ flag=KR }:: Südkorea | ::{ flag=BR }:: Brasilien | ::{ flag=US }:: Vereinigte Staaten |
 | [osu!mania 7K World Cup 2023](MWC/2023_7K) | 15.12.2022 | 18.02.2023 | ::{ flag=PH }:: Philippinen | ::{ flag=KR }:: Südkorea | ::{ flag=MY }:: Malaysia |
 | [osu!mania 4K World Cup 2023](MWC/2023_4K) | 13.07.2023 | 24.09.2023 | ::{ flag=US }:: Vereinigte Staaten | ::{ flag=KR }:: Südkorea | ::{ flag=ID }:: Indonesien |
-| [osu!mania 7K World Cup 2024](MWC/2024_7K) | 15.12.2023 | 25.02.2024 | *TBD* | *TBD* | *TBD* |
+| [osu!mania 7K World Cup 2024](MWC/2024_7K) | 15.12.2023 | 25.02.2024 | ::{ flag=KR }:: South Korea | ::{ flag=MY }:: Malaysia | ::{ flag=PH }:: Philippines |
```
Comment (FreezyLemon) [inline wiki/Tournaments/de.md:87]: All other country names are translated, so the new ones should be, too.

**Review by FreezyLemon — CHANGES_REQUESTED** [review]
Only one small issue.

Out of curiosity, how did you figure out what was changed in the English version? The osu-wiki-tools diff tracker seems to be incorrect for this article, maybe because of an incorrect `outdated_since`.

File: wiki/Tournaments/de.md (hunk 3 above)
Comment (The-Last-Cookie) [reply wiki/Tournaments/de.md:87]: This is the reason why it needs a second pair of eyes.

~~One could say that I did this on purpose for you to find something haha~~

```suggestion
| [osu!mania 7K World Cup 2024](MWC/2024_7K) | 15.12.2023 | 25.02.2024 | ::{ flag=KR }:: Südkorea | ::{ flag=MY }:: Malaysia | ::{ flag=PH }:: Philippinen |
```

**Comment by The-Last-Cookie** [comment]
> Out of curiosity, how did you figure out what was changed in the English version? The osu-wiki-tools diff tracker seems to be incorrect for this article, maybe because of an incorrect `outdated_since`.

I always use the [osu! wiki status](https://osu.wiki/status/de) page. In this particular instance, it's this view here: https://osu.wiki/status/diff/9e7171bcffbb05be58c6444cf46ea6ba11e1a59c/wiki/Tournaments/en.md

**Comment by Walavouchey** [comment]
> The osu-wiki-tools diff tracker seems to be incorrect for this article, maybe because of an incorrect `outdated_since`.

[osu-wiki-tools](https://github.com/walavouchey/osu-wiki-tools) doesn't have a diff tracker... ?

**Comment by FreezyLemon** [comment]
> [osu-wiki-tools](https://github.com/walavouchey/osu-wiki-tools) doesn't have a diff tracker... ?

I meant osu-wiki-status. 

The diff that was shown to me was slightly incorrect and also out of sync with the changes in this PR, that's why I asked.

**Review by FreezyLemon — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**